common/djangoapps/course_modes/models.py

```diff
@@ -87,7 +87,14 @@ class CourseMode(models.Model):
     # The system prefers to set this automatically based on default settings. But
     # if the field is set manually we want a way to indicate that so we don't
     # overwrite the manual setting of the field.
-    expiration_datetime_is_explicit = models.BooleanField(default=False)
+    expiration_datetime_is_explicit = models.BooleanField(
+        default=False,
+        verbose_name=_("Upgrade Deadline Explicitly Defined"),
+        help_text=_(
+            "OPTIONAL: Set to True if the upgrade deadline date is explicitly defined. "
+            "Set to False if there is no upgrade deadline or to use the default upgrade deadline."
```

Comment:
By "default upgrade deadline" do you mean the deadline that has been entered in Studio?

Reply:
I was thinking default deadline is the deadline automatically calculated by Studio (10 days prior to the end date).